website/blog/2021-02-12-introducing-oclif-core.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Introducing @oclif/core
+---
+Greetings!
+
+We hope this blog post finds you well.
+
+### Introducing... 
+
+We are excited to announce the next iteration of the oclif project today: `@oclif/core`.
+
+We have learned a lot in the last three years of developing oclif and developing on oclif, running millions of commands a day via Heroku & Salesforce CLIs.
+`@oclif/core` ("Core") simplifies the development experience and introduces highly requested new features.
+
+Core combines the essential oclif packages into one "core" package, aptly named `@oclif/core`. 
+
+Core also introduces:
+- A default command option
+- Use colon or spaced command syntax
+- Async parsing 
+- Command piping to arguments
+
+With default command functionality, Core simplifies the oclif project and removes the notion of single vs multi command CLIs. Core CLIs can have 1 or many commands.
+
+Along with Core, we have combined the oclif and oclif-dev CLIs into a single CLI with a new AWS S3 compatible publishing scheme.
+
+### What to expect in the near future
+
+Core is in pre-release beta and being actively developed for new internal Salesforce CLIs.
+
+Much documentation needs to be written in the comming months including migration paths. Look for forthcoming blog posts and documentation on [oclif.io](https://oclif.io).
+
+I've written Core with current CLIs migration in mind. Migration onto it should be as painless as possible with many exports remaining entirely unchanged.
```

Comment:
You use "I" as the pronoun here but sign it "the oclif team". I would change this to "we" or change the signature to your name.